### ISSUE TITLE
Typo in trilerp function

### ISSUE THREAD

**Issue by szellmann** [RayTracing/raytracing.github.io#271]
One very minor issue: in the listing right below Fig. 5-4 in the "next week" book:
```inline float trilinear_interp(float c[2][2][2], float u, float v, float w) {
    float accum = 0;
    for (int i=0; i < 2; 1++)
        for (int j=0; j < 2; j++)
            for (int k=0; k < 2; k++)
                accum += (i*u + (1-i)*(1-u))*
                         (j*v + (1-j)*(1-v))*
                         (k*w + (1-k)*(1-w))*c[i][j][k];

    return accum;
}
```
In the outer loop it presumably should be `i++` rather than `1++`. That's in the new html version of the book, in the original pdf version this seems to be correct.

**Comment by szellmann** [comment]
I just saw that I can fix it myself. See https://github.com/RayTracing/raytracing.github.io/pull/272

**Comment by hollasch** [comment]
Fixed in #280, development branch.
